integrations/hexo-ariada/lib/hexo-ariada.js

```
@@ -0,0 +1,199 @@
+// SPDX-FileCopyrightText: 2026 Agonist Development AB
+// SPDX-License-Identifier: EUPL-1.2
+'use strict';
+
+const { createReadStream } = require('node:fs');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)